UpExtreme/mockup-device-1amp.asl

```diff
@@ -0,0 +1,40 @@
+ /** @file
+  The definition block in ACPI table for RT5682 under SoundWire Controller
+  (link2)
+
+  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+**/
+
+DefinitionBlock (
+  "",
+  "SSDT",
+   2,
+  "INTEL",
+  "RTK0Tabl",
```

Comment:
INTTabl? I have no idea what is the suitable name.

Reply:
I chose CML1 for 1-amp